elasticdl/python/data/reader/csv_reader.py

```diff
@@ -0,0 +1,70 @@
+import csv
+
+import numpy as np
+import tensorflow as tf
+
+from elasticdl.python.data.reader.data_reader import (
+    AbstractDataReader,
+    Metadata,
+)
+
+
+class CSVDataReader(AbstractDataReader):
+    """This reader is used to read data from a csv file. It is convenient for
+    user to locally run and debug a Keras model by using this reader.
+    However, it can not be used with distribution strategy because it can not
+    read data by line indices.
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Args:
+            kwargs should contains "seq" and "columns" like
+            'seq=",",column=["sepal.length", "sepal.width", "variety"]'
+        """
+        AbstractDataReader.__init__(self, **kwargs)
```

Comment:
Should we add check_required_kwargs here?

Reply:
I have set the default value for arguments in `__init__`, so i think we don't need to check.